src/Promotions/InPluginUpsells/resources/js/components/FreeAddOnTab.js

```diff
@@ -0,0 +1,102 @@
+import {Hero} from './Hero';
+import {Card} from './Card';
+import {transformStrong} from '../utils';
+
+const highlights = [
+    {
+        icon: 'https://givewp.com/downloads/upsells/images/time-icon.svg',
+        text: 'Flexible Delivery Times',
+    },
+    {
+        icon: 'https://givewp.com/downloads/upsells/images/trend-icon.svg',
+        text: 'Pack your Reports with Stats',
+    },
+    {
+        icon: 'https://givewp.com/downloads/upsells/images/people-icon.svg',
+        text: 'Customize the Content & Recipients',
+    },
+];
+
+const {heading, description, reports, form} = window.GiveAddons.freeAddon;
+const {siteUrl, siteName} = window.GiveAddons;
+
+export const FreeAddOnTab = () => {
+    const {userSubscribed, hasSubscriptionError, subscribeUser} = useFreeAddonSubscription();
+    const firstNameInput = useRef();
+    const emailInput = useRef();
+
+    const handleSubscribe = async (event) => {
+        event.preventDefault();
+
+        await subscribeUser(firstNameInput.current.value, emailInput.current.value, siteUrl, siteName);
+    };
+
+    return (
+        <article className={styles.freeAddonSection}>
+            <Hero heading={heading} description={description} />
+            <Card as="article" className={cx(styles.card, styles.emailReports)}>
+                <div>
+                    <div className={styles.nameAndFlag}>
+                        <h3 className={styles.title}>
+                            <img className={styles.icon} src={reports.icon} alt={reports.heading} />
+                            {reports.heading}
+                        </h3>
+                    </div>
+                    <div className={styles.description}>
+                        {reports.description.map((text) => (
+                            <p dangerouslySetInnerHTML={{__html: transformStrong(text)}}></p>
+                        ))}
+                    </div>
+                </div>
+                <aside className={styles.includes}>
+                    <h4>{reports.highlights.heading}</h4>
+                    {reports.highlights.items.map(({icon, text}) => (
+                        <div className={styles.nameAndFlag}>
+                            <img className={styles.icon} src={icon} alt="Icon" />
+                            <strong>{text}</strong>
+                        </div>
+                    ))}
+                </aside>
+            </Card>
+            <Hero heading={form.heading} className={styles.informationHero} />
+            <Card className={cx(styles.card, styles.formCard)}>
+                {userSubscribed ? (
+                    <span className={styles.formCard__thankYou}>{form.submissionConfirmation}</span>
+                ) : (
+                    <>
+                        {hasSubscriptionError && (
+                            <div className={styles.submissionError}>
+                                There was an issue submitting your information. Please try again. If the problem
```

Comment:
@JasonTheAdams can we make these strings translatable?

Reply:
There's not really any point. Literally every other string on this page is only ever going to be in English, unless we start providing our own translations on our server.